### PR TITLE
#34 Update logic for no result

### PR DESCRIPTION
**Pull request by InscribeDeeper**
To avoid an infinite loop if there is no result on the first time search

### CODE REVIEW

File: googlesearch/__init__.py [hunk 1]
```diff
@@ -72,3 +72,7 @@ def search(term, num_results=10, lang="en", proxy=None, advanced=False, sleep_in
                     else:
                         yield link["href"]
         sleep(sleep_interval)
+
+        if start == 0:
+            print("No result found with this search term", escaped_term)
```
Comment (Nv7-GitHub) [inline googlesearch/__init__.py:77]: Please return an empty list or throw an error instead, unintended logging can be annoying

**Review by Nv7-GitHub — CHANGES_REQUESTED** [review]
Thank you for the PR!

File: googlesearch/__init__.py (hunk 1 above)
Comment (InscribeDeeper) [reply googlesearch/__init__.py]: Thanks for checking. Updated

**Comment by InscribeDeeper** [comment]
> Seems good! This doesn't get affected by `yield`, right?

Yeah. If it goes to `yield`, the start index will definitely be larger than 0. I tested with this query: `search("site:https://www.rockwellautomation.com+rockwell")`

**Comment by Nv7-GitHub** [comment]
Ok, I'll include this in the next release!

**Pull request merged**